tests/datatypes/test_list.py

```diff
@@ -686,7 +686,6 @@ class InplaceListOperationTests(InplaceOperationTestCase, TranspileTestCase):
         'test_add_frozenset',
         'test_add_range',
         'test_add_set',
```

Comment:
I did not remove this test because it incorrectly relies on the Python and Java output for sets to be in the same order